rewrites.js

```diff
@@ -176,6 +176,11 @@ exports.REWRITES = [
     source: '/:collectiveSlug/:verb(tiers|contribute|events|projects|connected-collectives)',
     destination: '/contribute',
   },
+  {
+    source:
+      '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:collectiveSlug/:verb(tiers|contribute|events|projects|connected-collectives)',
```

Comment:
Events & project can't have children

```suggestion
      '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:collectiveSlug/:verb(tiers|contribute|connected-collectives)',
```